unblob/logging.py

```diff
@@ -1,4 +1,5 @@
 import logging
+import logging.config
```

Comment:
unused?

Reply:
weirdly enough, this import made `logging.handlers.WatchedFileHandler` work while I should have been importing with `from logging.handlers import WatchedFileHandler`.

Fixed the import, allowing me to remove the `# type: ignore` next to `WatchedFileHandler`.